### ISSUE TITLE
GraphQL types not updated when adding collection-link

### ISSUE THREAD

**Issue by Fen747** [kaviarjs/x-bundle#3]
**Describe the bug**
When adding a collection link, the `{entity}.links.ts` file is correctly updated, but the GQL definition is not

**To Reproduce**
`x`
`x:collection-link`
`go through the steps to define a link`
`validate`

**Expected behavior**
The link should be represented in the GraphQL definition, but is still missing

**Screenshots**
<img width="672" alt="Screenshot 2021-01-08 at 13 08 23" src="https://user-images.githubusercontent.com/13072245/104013997-97ef4100-51b2-11eb-97ee-050124316574.png">
<img width="466" alt="Screenshot 2021-01-08 at 13 08 47" src="https://user-images.githubusercontent.com/13072245/104014017-a63d5d00-51b2-11eb-813f-fb4348446307.png">

**Desktop (please complete the following information):**
 - OS: MacOs
 - node 14

**Comment by theodorDiaconu** [comment]
Yes, currently we don't support dynamic updating. But this is possible and fairly easy to be done.

**Comment by Fen747** [comment]
Oh okauy, actually, the following warning : "Please ensure that your database model files and GraphQL types are updated accordingly." let me think I should just check that it has been correctly done by X-Way ;)

**Comment by theodorDiaconu** [comment]
Got it. A descriptive message what the user should do makes a lot of sense.
